ansible/roles_ocp_workloads/ocp4_workload_fuse_online/tasks/pre_workload.yml

```diff
@@ -4,7 +4,7 @@
   k8s:
     state: present
     definition: "{{ lookup('template', role_path ~ '/templates/clusterresourcequota.j2' ) | from_yaml }}"
-  when: ocp4_workload_fuse_online_user_needs_quota|d(False)|bool
+  when: ocp4_workload_fuse_online_user_needs_quota|d(false)|bool
```

Comment:
Let's go ahead and clean this up. `d(...)` should change to `default(...)` and add spaces around the `|` filter operator.